locale/en/corteza-webapp-compose/module.yaml

```diff
@@ -165,6 +165,25 @@ edit:
           label: Soft duplicate value validation
           description: Record will be saved and user will be presented with a warning when a duplicate value is detected in the selected fields in any existing record of this module
 
+    uniqueValues:
+      title: Unique values
+      allowRecordsSave: Allow record saving if duplicate values are found
+      warningLabel: warning or error message toast when constraint matches
```

Comment:
Capitalize it, and let's change it to "Warning or error message displayed when constraint matches".
Toast doesn't mean anything to users